### PR TITLE
3610 bag export rest

### PR DESCRIPTION
**Pull request by leventegal-she**
Closes https://github.com/hzi-braunschweig/SORMAS-Project/issues/3610

### CODE REVIEW

**Comment by HolgerReiseVSys** [comment]
@leventegal-she Sorry, I cannot do the review atm

File: sormas-backend/src/main/java/de/symeda/sormas/backend/bagexport/BAGExportFacadeEjb.java [hunk 1]
```diff
@@ -62,6 +64,9 @@
 	@PersistenceContext(unitName = ModelConstants.PERSISTENCE_UNIT_NAME)
 	private EntityManager em;
 
+	@EJB
+	private ConfigFacadeEjbLocal configFacade;
```
Comment (vidi42) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/bagexport/BAGExportFacadeEjb.java:68]: The field doesn't seem to be used

File: sormas-api/src/main/java/de/symeda/sormas/api/user/UserRole.java [hunk 2]
```diff
@@ -61,7 +61,8 @@
 	IMPORT_USER(false, false, false, false, JurisdictionLevel.NONE),
 	REST_EXTERNAL_VISITS_USER(false, false, false, false, JurisdictionLevel.NONE),
 	REST_USER(false, false, false, false, JurisdictionLevel.NONE),
-	SORMAS_TO_SORMAS_CLIENT(false, false, false, false, JurisdictionLevel.NONE);
+	SORMAS_TO_SORMAS_CLIENT(false, false, false, false, JurisdictionLevel.NONE),
+	BAG_USER(false, false, false, false, JurisdictionLevel.NONE);
```
Comment (vidi42) [inline sormas-api/src/main/java/de/symeda/sormas/api/user/UserRole.java:65]: Please add the _BAG_USER constant bellow to be used in the @RolesAllowed annotation
Also add it to the `addAssignableRoles` method

File: sormas-backend/src/main/java/de/symeda/sormas/backend/bagexport/BAGExportFacadeEjb.java (hunk 1 above)
Comment (vidi42) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/bagexport/BAGExportFacadeEjb.java:68]: The field doesn't seem to be used

**Review by vidi42 — CHANGES_REQUESTED** [review]

**Review by vidi42 — APPROVED** [review]

**Pull request merged**